strawberry_django/mutations/resolvers.py

```diff
@@ -54,8 +54,7 @@
 
 
 def _parse_pk(
-    value: ParsedObject | strawberry.ID | _M | None,
-    model: type[_M],
+    value: ParsedObject | strawberry.ID | _M | None, model: type[_M], key_attr: str
```

Comment:
Give it a default value to avoid it being required

```suggestion
    value: ParsedObject | strawberry.ID | _M | None,
    model: type[_M],
    *,
    key_attr: str = "id",
```